apps/condo/domains/user/schema/SendMessageToSupportService.test.js

```diff
@@ -125,6 +126,40 @@ describe('SendMessageToSupportService', () => {
         expect(messages[0].meta.organizationsData).toHaveLength(2)
     })
 
+    test('with organization and service consumer', async () => {
+        const userClient = await makeClientWithProperty()
+        const adminClient = await makeLoggedInAdminClient()
+        const [organization] = await registerNewOrganization(userClient)
+
+        const unitName = faker.random.alphaNumeric(8)
+        const [resident] = await createTestResident(adminClient, userClient.user, userClient.property, {
+            unitName,
+        })
+        const accountNumber1 = faker.random.alphaNumeric(8)
+        await createTestServiceConsumer(adminClient, resident, organization, {
+            accountNumber: accountNumber1,
+        })
+        await addResidentAccess(userClient.user)
+
+        const payload = {
+            text: `Test message from resident to support. This message should be sent from ${EMAIL_FROM}. Resident must be attached to two organizations.`,
```

Comment:
I think you should use `faker` package for this string generation. For example `faker.lorem.paragraph()`

Reply:
Here, I followed the pattern of the rest of the email texts in the tests. But I forgot to change the text for the new case. The text has now been changed to be more appropriate.